.travis.yml

```diff
@@ -1,20 +1,35 @@
+sudo: false
```

Comment:
`sudo` can be removed, it no longer has any effect:

* https://blog.travis-ci.com/2019-10-07-multi-cpu-architecture-support

Reply:
Thanks!